AFQ/utils/tests/test_volume.py

```diff
@@ -1,7 +1,15 @@
 import numpy as np
+import numpy.testing as npt
 import pytest
+import os.path as op
+
+import nibabel as nib
+
+from dipy.io.stateful_tractogram import Space
+from dipy.io.streamline import StatefulTractogram
 
 import AFQ.utils.volume as AFV
+import AFQ.data as AFD
```

Comment:
These should be  imported as lower-case (sorry for missing this before!)

```suggestion
import AFQ.utils.volume as afv
import AFQ.data as afd
```